app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/survey/SurveyQuestionScreen.kt

```diff
@@ -50,8 +51,11 @@ fun SurveyQuestionScreen(
                 DROPDOWN -> SurveyDropDownQuestion(answers = surveyQuestion.answers.sortedByDisplayOrder()) {
                     onChooseAnswer(surveyQuestion.id, it)
                 }
-                SMILEY -> if (surveyQuestion.answers.size >= NUMBER_OF_SMILEY_ANSWERS) {
-                    SurveySmileyQuestion(answers = surveyQuestion.answers.sortedByDisplayOrder()) {
+                SMILEY, THUMBS -> if (surveyQuestion.answers.size >= NUMBER_OF_EMOJI_ANSWERS) {
```

Comment:
Optional: for more cases, it can grow longer and reach max line length, which requires us to use line breaks. I think it's better to use line breaks for all the cases irrespective of the number of them. What do you think?
```suggestion
                SMILEY,
                THUMBS -> if (surveyQuestion.answers.size >= NUMBER_OF_EMOJI_ANSWERS) {
```